qutip/control/optimizer.py

```diff
@@ -179,15 +179,17 @@ class Optimizer(object):
         to calculate approximate gradients
         Note it is set True automatically when the alg is CRAB
 
-    amp_lbound : float or list of floats
+    amp_lbound : float or list of floats or ndarray of floats
         lower boundaries for the control amplitudes
         Can be a scalar value applied to all controls
         or a list of bounds for each control
+        or an ndarray of bounds for each control at each time slot
```

Comment:
Apologies for the drive by reviewing (and please feel free to ignore!): It seems odd for a list and a one dimensional ndarray to behave differently. Would it be better if the list and a one dimensional ndarray behaved in the same way and only a two dimensional ndarray triggered the new behaviour?

I don't know how this function is typically used though, so I could easily be wrong.

Reply:
Yes you are right, I made the changes.